sockshop/payment/test/unit/service_unit_test.go

```diff
@@ -1,6 +1,7 @@
-package payment
+package payment_unit_test
 
 import (
+	. "github.com/weaveworks/microservices-demo/sockshop/payment"
```

Comment:
Unit tests should definitely live alongside the code under test. This file should move back to sockshop/payment, and…